index.html

```diff
@@ -0,0 +1,181 @@
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+        <div class="work-tonic">
+            <div>
+                <img src="images/Snapshoot3.png" alt="Snapshoot3">
+            </div>
+            <h1>Tonic</h1>
+            <div class="text-info">
+                <ul>
+                <li class="client-name">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+        <div class="work-tonic">
+            <div>
+                <img src="images/Snapshoot4.png" alt="snapshoot4">
+            </div>
+            <h1>Multi-Post
+                Stories</h1>
+            <div class="text-info">
+                <ul>
+                <li class="client-name">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+            </ul>
+            </div>
+            <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <button>HTML</button>
+                <button>CSS</button>
+                <button>javaScript</button>
+            </div>
+            <button class="see-proj">See project</button>
+        </div>
+    </section>
+                                 <!-- start About me -->
+    <section class="about-me">
+        
+        <div class="about-me-text">
+            <h2>About
+                 Myself</h2> 
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature
+                     or website Look through some of my work and experience! If you like what 
+                     you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+        </div>
+        <div class="hero-socialmedia-icons">
+            <ul>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/linkdin.png" alt="linkdin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+        </ul>
```

Comment:
- [ ] Please make sure the button to get my resume doesn't have the text underlined and the social icons are properly aligned, as per design.
![](https://i.imgur.com/6NmquwG.png)